README.rst

```diff
@@ -23,11 +23,16 @@ Usage
     tutor plugins enable oars
     tutor plugins enable ralph
 
-2. Save the changes to the environment::
+2. Optionally, you can allow the LRS API to accesible by domain by setting the following variables in you **config.yml**, the host will default to **ralph.{{LMS_HOST}}**:
```

Comment:
Can you update this to be clear that this is only necessary for `tutor local`? In `tutor dev` it should be always available.

Reply:
Ready